src/NativeScript/ObjC/Inheritance/ObjCClassBuilder.mm

```diff
@@ -30,13 +30,12 @@ @protocol TNSDerivedClass
 using namespace JSC;
 using namespace Metadata;
 
-static WTF::CString generateAnonymousSubclassName(const char* className) {
-    int i = 0;
-    WTF::CString anonymousName;
+static WTF::CString generateSafeSubclassName(const char* className) {
+    WTF::CString anonymousName(className);
 
-    do {
-        anonymousName = WTF::String::format("%s%d", className, ++i).utf8();
-    } while (objc_getClass(anonymousName.data()));
```

Comment:
I believe its intent is clear enough, although I admit I am not unbiased in this. Maybe a small comment that explains what's a safe subclass name and why we need it could help, but I'm only offering this as a suggestion.